test/contracts/src/AccessListHelper.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract AccessListHelper {
+    uint256 private storedValue;
```

Comment:
I think this should be a mapping, accessing the same value multiple times is not useful for checking the AccessList usage. The value is considered `warm` after the first read, and subsequent reads should be cheaper.